doc/reference/reference_lua/box_space/user_defined.rst

```diff
@@ -1,44 +1,33 @@
 .. _box_space-user_defined:
 
 ===============================================================================
-space_object:user_defined()
+space_object extensions
 ===============================================================================
 
-.. class:: space_object
+You can extend ``space_object`` with custom functions as follows:
 
-    .. method:: user_defined()
+(1) Create a Lua function.
+(2) Add the function name to a predefined global variable, which has the ``table`` type.
```

Comment:
The structure looks awkward with only one "predefined global variable" below.
Also, please fix the step number formatting.